pkg/plugin/manager.go

```diff
@@ -82,10 +82,13 @@ type Client interface {
 type ManagerStore interface {
 	Store(name string, client Client, metadata *Metadata, cmd string) error
 	StoreJS(name string, jspc JSPlugin) error
+	Get(name string) (Client, bool)
 	GetJS(name string) (JSPlugin, bool)
+	Remove(name string)
```

Comment:
If a plugin was delete while the user was looking at a view, what should happen?

Currently https://github.com/vmware-tanzu/octant/blob/master/internal/api/content_manager.go#L191 returns the error each poll, but we might want to redirect back to a 404-Like page